sdk/src/store.rs

```diff
@@ -3725,6 +3801,29 @@ pub mod tests {
         let store = Store::load_from_asset(&ap, true, &mut report).expect("load_from_asset");
         println!("store = {store}");
     }
+
+    /*
+    #[test]
```

Comment:
Did you mean to leave it commented out. If so please add a comment saying why

Reply:
The test files are from MS.  We need to get files we can ship.